api/filter/accesslog/accesslog.proto

```diff
@@ -50,26 +50,52 @@ message AccessLogCommon {
   // downstream byte received (i.e. time it takes to receive a request).
   google.protobuf.Duration time_to_last_rx_byte = 6;
 
+  // Interval between the first downstream byte received and the first upstream byte sent. There may
+  // by considerable delta between *time_to_last_rx_byte* and this value due to filters.
+  // Additionally, the same caveats apply as documented in *time_to_last_downstream_tx_byte* about
```

Comment:
Nit: a style that I think is nice is to use a ref link to the field, as this can be useful when the proto grows larger. In this case it's probably not a big win, but just throwing it out there.

Reply:
We aren't emitting docs yet for this since it's part of an API. My plan was to go back through and figure out how to emit API docs in a way that is not confusing (possibly by splitting into a different file also). OK If I deal with this in a follow up?